src/analysis/classic/hunter/modules/features/CooldownThroughputTracker.ts

```diff
@@ -0,0 +1,17 @@
+import SPELLS from 'common/SPELLS/classic';
+import CoreCooldownThroughputTracker, {
+  BUILT_IN_SUMMARY_TYPES,
+} from 'parser/shared/modules/CooldownThroughputTracker';
+
+class CooldownThroughputTracker extends CoreCooldownThroughputTracker {
+  static castCooldowns = [
+    ...CoreCooldownThroughputTracker.castCooldowns,
+    // Add Cooldown Spells specific to Spec
+    {
+      spell: SPELLS.ICY_VEINS.id,
+      summary: [BUILT_IN_SUMMARY_TYPES.DAMAGE],
+    },
```

Comment:
This is a mage spell. Should be removed.

Reply:
Done, Not sure where this is showing up in the UI/Report